src/components/CameraError/index.js

```diff
@@ -58,13 +58,9 @@ export default class CameraError extends Component<Props, State> {
           onDismiss={this.handleDismiss}
           renderInstruction={ str => parseTags(str,
             ({text}) =>
-            <button
-              type="button"
-              onClick={this.handleFallbackClick}
-              className={style.fallbackLink}
-            >
+            <a onClick={this.handleFallbackLinkClick} className={style.fallbackLink} >
```

Comment:
This a tag has not href therefore it will be unreachable using the keyboard, which is a regression compared to latest.

Reply:
I'll update it to use the original `<button>` element and rename the class name to reflect what it is instead. Simpler overall.